src/libraries/Microsoft.PowerFx.Interpreter/RecalcEngine.cs

```diff
@@ -32,15 +33,15 @@ public class RecalcEngine : IScope
 
         internal Dictionary<string, RecalcFormulaInfo> Formulas { get; } = new Dictionary<string, RecalcFormulaInfo>();
 
-        private readonly CultureInfo _cultureInfo;
+        PowerFxConfig _powerFxConfig;
```

Comment:
>PowerFxConfig [](http://example.com/codeflow?start=8&length=13)

Design discussion: what should go on Config vs. on this class directly?
Ie, EnumStore can't be static anymore and needs to live somewhere.  We'll have a similar issue coming with BuiltinFunctions. We have a custom function lists here (line 32).

Reply:
The extra functions can be added to the Config instead, either in this PR or in a future PR (when built-in functions are also made non-static)